picamera2/encoders/jpeg_encoder.py

```diff
@@ -4,11 +4,13 @@
 
 
 class JpegEncoder(MultiEncoder):
-    def __init__(self, num_threads=4, q=85, colour_space='RGBX'):
+    def __init__(self, num_threads=4, q=85, colour_space="RGBX"):
         super().__init__(num_threads=num_threads)
         self.q = q
         self.colour_space = colour_space
 
     def encode_func(self, request, name):
         array = request.make_array(name)
-        return simplejpeg.encode_jpeg(array, quality=self.q, colorspace=self.colour_space)
+        return simplejpeg.encode_jpeg(
+            array, quality=self.q, colorspace=self.colour_space
+        )
```

Comment:
Not sure how I feel about the formatting here and in all the other places where the function arguments are put on the next line.  The original reads easier to me.